contrib/docs/tutorials/help-im-behind-a-corporate-proxy.md

```diff
@@ -56,3 +58,29 @@ if (process.env.HTTPS_PROXY) {
 ```
 
 4. Start the backend with `yarn start`
+
+## config
+
+If your development environnement is in the cloud (like with [AWS Cloud9](https://aws.amazon.com/cloud9/) or an instance of [Theia](https://theia-ide.org/)), you will need to update your configuration.
+
+In `app.local.config`, you will probably need to make some changes, the exact values will depend of your setup.
```

Comment:
```suggestion
In `app-config.local.yaml`, you will probably need to make some changes, the exact values will depend on your setup.
```

That's the config you are referring to right?

Reply:
Of course yes